tests/test_mlflow_integration.py

```diff
@@ -0,0 +1,154 @@
+import os
+import subprocess
+import sys
+
+import pandas as pd
+import pytest
+from sklearn import datasets
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+import mlflow
+
+
+@pytest.fixture(scope="module")
+def setup_servers():
+    with subprocess.Popen([sys.executable, "-m", "mlflow", "server", "--port", "5000"]) as proc:
+        try:
+
+            mlflow_tracking_url = "http://127.0.0.1:5000"
+            uc_oss_url = "uc:http://127.0.0.1:8080"
+
+            mlflow.set_tracking_uri(mlflow_tracking_url)
+            mlflow.set_registry_uri(uc_oss_url)
+
+            yield mlflow_tracking_url
+        finally:
+            proc.terminate()
+
+
+def test_integration(setup_servers):
+    catalog = "unity"
+    schema = "default"
+    registered_model_name = "iris"
+    model_name = f"{catalog}.{schema}.{registered_model_name}"
+    mlflow.set_experiment("iris-uc-oss")
+    try:
+        mlflow.MlflowClient().get_registered_model(model_name)
+    except Exception as e:
+        e.args[0].startswith("NOT_FOUND")
+    else:
+        assert False, "Expected exception for missing model not raised"
```

Comment:
```suggestion
        with pytest.raises(MlflowException, match="NOT_FOUND"):
            mlflow.MlflowClient().get_registered_model(model_name)
```
nit for consistency for our other tests

Reply:
(not sure if that's the right Exception - looks like it might be the UC proto error that's passed through?)

Reply:
Yes.  The UC exception is whats returned afaict.